### DISCUSSION TITLE
@astrojs/code-snippets package

### DISCUSSION THREAD

**Discussion by NEWESTERS** [withastro/docs#2885] [Ideas]
I want to mark diff in my code snippets like in astro docs website.

As i see there is plugin [@astrojs/code-snippets](https://github.com/withastro/docs/blob/main/integrations/astro-code-snippets.ts) implemented inside docs repository. Is there any plans to publish it separately?

**Comment by yanthomasdev** [comment]
> As i see there is plugin [@astrojs/code-snippets](https://github.com/withastro/docs/blob/main/integrations/astro-code-snippets.ts) implemented inside docs repository. Is there any plans to publish it separately?

The maintainer Hippo, who originally made the PR adding this package is working in an improved, public, version of this package that should be available soon. 🙌
